app/views/shared/_govuk_pagination_navigation.html.erb

```diff
@@ -0,0 +1,40 @@
+<nav class="govuk-pagination" role="navigation" aria-label="results">
+  <%- if pagy.prev %>
+  <div class="govuk-pagination__prev">
+    <a class="govuk-link govuk-pagination__link" href="<%= pagy_prev_url(pagy) %>" rel="prev">
+      <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+        <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+      </svg>
+      <span class="govuk-pagination__link-title">Previous</span>
+    </a>
+  </div>
```

Comment:
Can we use the GOVUK Components pagination components? https://govuk-components.netlify.app/components/pagination/

Reply:
I have now added the GOVUK component